src/components/dialogBox.jsx

```diff
@@ -0,0 +1,204 @@
+import React from "react";
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import CodeIcon from '@material-ui/icons/Code';
+import CloseIcon from '@material-ui/icons/Close';
+import TextField from '@material-ui/core/TextField';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import FilterNoneOutlinedIcon from '@material-ui/icons/FilterNoneOutlined';
+import useClipboard from "react-use-clipboard";
+import { handleShareToTwitterClick } from "../utils";
+import { Snackbar } from "@material-ui/core";
+import Alert from "@material-ui/lab/Alert";
+import DialogContentText from '@material-ui/core/DialogContentText';
+
+export const DialogBox = ({ repoList = ["apache/apisix"],}) => {
+  
+  const title = window.location.href.includes("contributorOverTime")
+    ? "contributor over time"
+    : "monthly active contributor"
+    console.log('shareText:',title)
+  const Url = window.location.href.split('repo=apache/apisix')
+  const Tail = Url[1]
+  const OverTimegetShareParams = () => `?chart=contributorOverTime&repo=${repoList.join(",")}${Tail}`
+  const MonthlyActivegetShareParams = () => `?chart=contributorMonthlyActivity&repo=${repoList.join(",")}${Tail}`;
+  const OverTimevalue = `<iframe style={{ width: "100%", height: "auto" , minWidth: "600px", minHeight: "1000px" }} src="https://git-contributor.com/${OverTimegetShareParams()}" frameBorder="0"></iframe>`
+  const MonthlyActivevalue = `<iframe style={{ width: "100%", height: "auto" , minWidth: "600px", minHeight: "1000px" }} src="https://git-contributor.com/${MonthlyActivegetShareParams()}" frameBorder="0"></iframe>`
+  const [, setEmOverTimecopy] = useClipboard(OverTimevalue, { successDuration: 3000 })
+  const [, setEmMonthlyActivecopy] = useClipboard(MonthlyActivevalue, { successDuration: 3000 })
+  const [, setMonthlyActiveCopied] = useClipboard(`https://git-contributor.com/${MonthlyActivegetShareParams()}`, { successDuration: 3000 });
+  const [, setOverTimeCopied] = useClipboard(`https://git-contributor.com/${OverTimegetShareParams()}`, { successDuration: 3000 });
+  const [openAlert, setOpenAlert] = React.useState(false);
+  const [open, setOpen] = React.useState(false);
+  const [fullWidth, ] = React.useState(true);
+  const [maxWidth, ] = React.useState('sm');
+
+  const SearchButton = () => (
+
+    <Button
+      color="secondary"
+      variant="contained"
+      value="Copy"
+      style={{ padding: '5px 20px', textTransform: 'none', position: 'absolute', bottom: '10px', right: '20px',backgroundColor:' #E53E3E'}}
+      onClick={handleiframeiframe}
+    >
+      Copy
+    </Button>
+  );
```

Comment:
```suggestion
    <Button
      color="secondary"
      variant="contained"
      value="Copy"
      style={{ padding: '5px 20px', textTransform: 'none', position: 'absolute', bottom: '10px', right: '20px',backgroundColor:' #E53E3E'}}
      onClick={handleiframeiframe}
    >
      Copy
    </Button>
  );
```
It's a good habit to format your code whenever you're done changing it